amy/autoemails/management/commands/repeated_jobs.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+import django_rq
+
+from autoemails.actions import (
+    BaseRepeatedAction,
+    InactiveProfileArchivalWarningRepeatedAction,
+    UpdateProfileReminderRepeatedAction,
+)
+from autoemails.models import RQJob, Trigger
+from autoemails.utils import check_status, scheduled_execution_time
+
+scheduler = django_rq.get_scheduler()
+logger = logging.getLogger("amy.signals")
+REPEATED_JOBS_BY_TRIGGER = {
+    "profile-update": UpdateProfileReminderRepeatedAction,
+    "profile-archival-warning-inactivity": InactiveProfileArchivalWarningRepeatedAction,
```

Comment:
Read the strings from `UpdateProfileReminderRepeatedAction` (class field to be added) and `InactiveProfileArchivalWarningRepeatedAction.ACTION_NAME`